log/log.go

```diff
@@ -55,6 +58,7 @@ func CleanExit(format string, v ...interface{}) {
 func Err(format string, v ...interface{}) {
 	fmt.Fprint(Stderr, pretty.Colorize("{{.Red}}[ERROR]{{.Default}} "))
 	fmt.Fprintf(Stderr, appendNewLine(format), v...)
+	ErrorState = true
```

Comment:
Given the general concurrency state of this entire package, it'd be sorta pedantic to require that this one variable be marked as not concurrency safe. After all... look at Stdout and Stderr.

I'm not opposed at all to the channel-based route. In fact, might there be a way to make it an int channel and pass in a specific error code? cli frustrates me in that it seems to only support returning `1`.